runway_model.py

```diff
@@ -31,45 +31,38 @@ def setup(opts):
     return model
 
 
-inputs = {"input_file" : runway.image}
-outputs = {"output_file" : runway.image}
+inputs = {"input_image" : runway.image}
+outputs = {"output_image" : runway.image}
```

Comment:
As a convention, we prefer not using the term "file" in the input name, as that appears in the UI and might be confusing (e.g. if I've selected any input other than "File", such as Camera)